src/obr/OpenFOAM/case.py

```diff
@@ -270,9 +267,33 @@ def has_openfoam_header(self, path: Path) -> bool:
     def _exec_operation(self, operation) -> Path:
         return logged_execute(operation, self.path, self.job.doc)
 
+    @property
+    def esi_version(self) -> bool:
+        """Check if esi version of OpenFOAM is sourced"""
+        wm_project_dir = os.environ["WM_PROJECT_DIR"]
+        return (Path(wm_project_dir) / "CONTRIBUTORS.md").exists()
+
     def decomposePar(self, args={}):
         """Sets decomposeParDict and calls decomposePar. If no decomposeParDict exists a new one
         gets created"""
+        tmp_zero = None
+        if not self.time_folder:
+            logging.warning(
+                f"No time folder found! Decomposition might lead to an unusable case."
+            )
+            zero_orig_path = Path(self.path / "0.orig")
```

Comment:
```suggestion
            zero_orig_path = self.path / "0.orig"
```

If `self.path` wasn't a `Path` to begin with, this wouldn't work. No explicit creation of a path object needed in that case.